src/Periphery/LiFiDEXAggregator.sol

```diff
@@ -69,7 +79,10 @@ contract LiFiDEXAggregator is WithdrawablePeriphery {
         address[] memory priviledgedUserList,
         address _owner
     ) WithdrawablePeriphery(_owner) {
-        bentoBox = IBentoBoxMinimal(_bentoBox);
+        if (_bentoBox == address(0) || _owner == address(0)) {
```

Comment:
this is incorrect. We currently deploy the LDA with _bentoBox == address(0) so this check (and the respective test) needs to be removed

Reply:
true! updated.